src/sotoki/utils/preparation.py

```diff
@@ -531,6 +538,34 @@ def merge_users_with_badges(
     logger.info("merged both sets")
     return users_with_badges
 
+def count_xml_rows(
+    src: pathlib.Path,
+    tag: str
+): 
+    """Count number of rows in an XML file having a given tag"""
+    args1= ["/usr/bin/env",
```

Comment:
Is there any reason to use `sed -n` here? I'd have thought `grep -F` would be faster.

Reply:
Nope, let's move to `grep -F` then

Reply:
Ah!